qa/nose_wrapper/__main__.py

```diff
@@ -1,4 +1,86 @@
 import sys
+
+if sys.version_info >= (3, 12):
```

Comment:
Do we need this in here and in the nose_utils? (I guess we do because nose_utils may be used in nose2 suites as well, but just double checking).